providers/cstor-operator/test.yml

```diff
@@ -18,254 +18,87 @@
           vars:
             status: 'SOT'
 
-
         - block:
-            - name: Downloading the csi-operator file
-              get_url:
-                url: "https://raw.githubusercontent.com/openebs/charts/master/archive/1.10.x/csi-operator-1.10.0.yaml"
-                dest: "{{ playbook_dir }}/csi-operator.yaml"
-                force: yes
-              register: csi_status
-              until:  "'OK' in csi_status.msg"
-              delay: 5
-              retries: 3
-              when: node_os == "ubuntu-16.04" or node_os == "centos"
-
-            - name: Downloading the csi-operator file
-              get_url:
-                url: "https://raw.githubusercontent.com/openebs/charts/master/archive/1.10.x/csi-operator-1.10.0-ubuntu-18.04.yaml"
-                dest: "{{ playbook_dir }}/csi-operator.yaml"
-                force: yes
-              register: csi_status
-              until:  "'OK' in csi_status.msg"
-              delay: 5
-              retries: 3
-              when: node_os == "ubuntu-18.04"
 
             - name: Downloading openebs cStor operator from charts
               get_url:
-                url: https://raw.githubusercontent.com/openebs/charts/master/archive/1.10.x/cstor-operator-1.10.0.yaml
+                url: https://raw.githubusercontent.com/openebs/charts/{{ release_branch }}/{{ release_version }}/cstor-operator-{{ release_tag }}.yaml

```

Comment:
Q:_ When it will be final build where no release_tag will be present as RC1 or RC2 , how this url will work then.
for e.g. release version is 2.4.0 final build then url will be like
`......./cstor-operator-.yaml` ?